backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/button/internal/BpkButtonImpl.kt

```diff
@@ -108,7 +111,12 @@ internal fun ButtonIcon(
 
 @Composable
 internal fun ButtonText(text: String, modifier: Modifier = Modifier) {
-  BpkText(text, modifier)
+  BpkText(
```

Comment:
I noticed that text wraps if button width is not big enought is some cases during the debugging